website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -45,11 +45,8 @@ resource "digitalocean_kubernetes_cluster" "foo" {
 }
 
 provider "kubernetes" {
-  host = "${digitalocean_kubernetes_cluster.foo.endpoint}"
-
-  client_certificate     = "${base64decode(digitalocean_kubernetes_cluster.foo.kube_config.0.client_certificate)}"
-  client_key             = "${base64decode(digitalocean_kubernetes_cluster.foo.kube_config.0.client_key)}"
-  cluster_ca_certificate = "${base64decode(digitalocean_kubernetes_cluster.foo.kube_config.0.cluster_ca_certificate)}"
+  host  = digitalocean_kubernetes_cluster.foo.endpoint
+  token = digitalocean_kubernetes_cluster.foo.kube_config[0].token"
```

Comment:
Leftover `"` at the end here.

Reply:
ACK, fixing as part of https://github.com/terraform-providers/terraform-provider-digitalocean/pull/315